src/test/resources/rename/test.soar

```diff
@@ -0,0 +1,21 @@
+source secondFile.soar
+
+# Even though the soarAgents.json file is badly formed, this file
+# should still work.
+
+set NGS_YES *YES*
+
+sp "elaborate*top-state
+    (state <s> ^superstate nil)
+-->
+    (<s> ^top-state $NGS_YES)
+"
+
+
+# Using Tcl exceptions is not common in Soar programs, but apparently
```

Comment:
None of what follows gets used by this particular test, so we might as well remove this too.